### ISSUE TITLE
Support ADB_INSTALL_TIMEOUT env variable

### ISSUE THREAD

**Issue by Kerem356** [MarathonLabs/marathon#398]
In our ci we use emulator without KVM, so i need a lot of time to install app on emulator. Around 13 minutes
For this case adb service has **ADB_INSTALL_TIMEOUT** environment variable.
If variable is set up - _adb install_ command execute fine
**_23:16:11  + adb -s emulator-5554 some.apk
23:29:05  Performing Streamed Install
23:29:05  Success_**

But when i use marathon - i got
**_com.malinskiy.marathon.android.exception.InstallException: com.android.ddmlib.InstallException: executeRemoteCommand timed out after 240000ms_**

I found some deault value here https://github.com/Malinskiy/marathon/blob/develop/vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/Const.kt#L3 but didn't find any change it

**Comment by Malinskiy** [comment]
Hey @Kerem356. Thanks for reporting this issue.

Can you please double check if this issue happens when you use the [vendor adam](https://github.com/Malinskiy/marathon/blob/a9737ea27b4f31d4d20ee5c1e2a6585d1b57724b/sample/android-app/Marathonfile#L16)?

Thanks

**Comment by Malinskiy** [comment]
Small update: new adam vendor implementation supports install timeout configuration

https://github.com/Malinskiy/marathon/blob/develop/vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/configuration/TimeoutConfiguration.kt

Will be available with the merge of #443 

If you want to use the envvar, marathon's YAML supports envvar interpolation:
```yaml
name: "my tests"
vendorConfiguration:
  type: "Android"
  #...
  timeoutConfiguration:
    pullFile: "${ADB_INSTALL_TIMEOUT}"
```
